generation/scrapePgnData.spec.ts

```diff
@@ -0,0 +1,19 @@
+import { updatePgnData, scrapePgnPage, pgnRootPage } from "./scrapePgnData"
+
+describe('PGN Data Scraper', () => {
+
+    it('should return data', async () => {
+        const pgns = await scrapePgnPage(pgnRootPage)
+        expect(pgns).toBeTruthy()
+        expect(Object.keys(pgns).length).toBeGreaterThan(0)
+    })
+
+    it('should save the PGN data to its file', async () => {
+        await updatePgnData()
+        expect(() => require('../src/PGNs.ts')).not.toThrow()
+        // eslint-disable-next-line @typescript-eslint/no-var-requires
+        const pgns = require('../src/PGNs.ts')
+        // make sure we parsed at least 2 pages
+        expect(pgns.length).toBeGreaterThan(200)
+    })
```

Comment:
@oliverrahner I think that a test is not the best place to run this update. Tests shouldn't have side effects, especially that might result in changes of committed files.

What do you think about adding a separate npm script to run scraping of PGNs?

Reply:
Agreed! Will start working on this. I have in the meantime already extended the scraping very much, so it makes even more sense.